### ISSUE TITLE
Build Teachers/Home tab

### ISSUE THREAD

**Issue by garettarrowood** [rubyforgood/stocks-in-the-future#42]
# Summary

The Rails4 boot on the private sif has more updated static pages that need to be added into this new iteration. This ticket represents copying over all the styles and copy of the Teachers / Home tab.

Page:
<img width="1680" alt="Screenshot 2023-08-02 at 10 31 49 AM" src="https://github.com/rubyforgood/stocks-in-the-future/assets/12770108/5de3a449-9a73-43de-8a8a-64d77597d3ec">

## Things to Consider

Reuse what Nav ViewComponents are applicable. Add new View Components for Teachers sub-nav.

# Criteria for Completion
 - [x] When a user clicks on "Teacher" in the main nav, the Apple/Book graphic moves off of "Home" and covers up "Teacher"
 - [x] The Teacher home page loads and looks like the screenshot - use styles and copy from deprecated repo
 - [x] This feature does not require login
 - [ ] Add any applicable tests

**Comment by garettarrowood** [comment]
Hey @naomiyocum ! This repo is going to start becoming more active in a week or so. Do you still want to put together this page, or have you moved on to other adventures?

**Comment by naomiyocum** [comment]
Hi Garett! Thank you for checking in. Unfortunately, I don't have as much time right now to finish up this branch within the next couple of weeks.

The branch I was working on is linked to this issue so the team can check out what I have coded so far. Here's some bullet points of what I completed:
- Create partial for mission/vision container (repeated on home/teacher/student pages)
- Create separate sub nav components (home and teacher)
- Create contact component to be shown on each page (contains SIF socials and address)
- Create teacher#index page

I was able to complete most of the tasks for this issue - just haven't gotten to testing. It's up to the team if they want to use it though!
